Jenkinsfile

```diff
@@ -79,9 +80,11 @@ pipeline {
     stage('Unit Tests') {
       steps {
         sh '''cd tests/
-              PYTHONPATH=../python/ pytest -s -vvv
+              PYTHONPATH=../python/ python3.5 -m pytest -s -vvv
+              PYTHONPATH=../python/ python2.7 -m pytest -s -vvv
```

Comment:
I'd like to only run the unit tests once, since they already take so long.   I think we can just require that kotekan unit tests need `python3`.